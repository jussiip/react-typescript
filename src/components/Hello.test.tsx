import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Hello } from './Hello';

it('renders correctly', () => {
  const props = {
    compiler: 'TypeScript',
    framework: 'React',
  };
  const tree = renderer.create(<Hello {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
