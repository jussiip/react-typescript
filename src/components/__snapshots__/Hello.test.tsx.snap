// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  Hello from 
  TypeScript
   and 
  React
  !
</h1>
`;
